test/quantization/test_quantize_script.py

```diff
@@ -1158,6 +1160,16 @@ def forward(self, x):
                    .check("CallMethod") \
                    .run(model.graph)
 
+    def test_qconfig_with_types(self):
+        data = [(torch.rand((1, 5), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
+        qconfig_dict = {torch.nn.Linear : default_qconfig, 'sub2.fc1' : default_per_channel_qconfig}
```

Comment:
Is this precedence also enforced for the eager API?

Reply:
I don't see the logic to enforce this in eager API

Reply:
Can you do that as a follow up PR so that eager and graph are consistent?

Reply:
yeah precedence is enforced here in eager: https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L41-L43